scene/3d/skeleton_3d.h

```diff
@@ -138,6 +155,16 @@ class Skeleton3D : public Node3D {
 
 	void _update_process_order();
 
+	// To process modifiers.
+	bool modifier_reset_on_save = true;
+	ModifierCallbackModeProcess modifier_callback_mode_process = MODIFIER_CALLBACK_MODE_PROCESS_IDLE;
+	Vector<ObjectID> modifiers;
```

Comment:
Better to use LocalVectors for anything internal since its much faster to access. We should also change other animation and Skeleton code to use this. Vector is more suited for passing information via function arguments and properties since its thread safe and refcounted, but this thread safety makes it slower for internal usage.